### ISSUE TITLE
1.21 "take_damage" experience source doesn't work

### ISSUE THREAD

**Issue by Sokario** [pufmat/skillsmod#99]
I was testing your [Take_Damage](https://puffish.net/skillsmod/docs/creators/configuration/experience-sources/built-in/take-damage) example on your wiki copy pasting it in game with the lastest mod version on 1.21 Minecraft.
However, players doesn't gain xp when taking damage.

Here is the experience.json file:
```
{
    "experience_per_level": {
        "type": "expression",
        "data": {
            "expression": "min(level + 10, 400)"
        }
    },
    "sources": [
        {
            "type": "puffish_skills:take_damage",
            "data": {
                "variables": {
                    "damage": {
                        "operations": [
                            {
                                "type": "get_taken_damage"
                            }
                        ]
                    }
                },
                "experience": [
                    {
                        "expression": "damage"
                    }
                ]
            }
        }
    ]
}
```

**Comment by pufmat** [comment]
I have successfully reproduced this. Thanks for reporting, it will be fixed soon.

**Comment by pufmat** [comment]
Fixed in version 0.13.2.
